lib/active_model_serializers/adapter/json_api/deserialization.rb

```diff
@@ -1,12 +1,12 @@
+require 'jsonapi'
```

Comment:
Isn't this gem already required elsewhere? I know we use it for the include directives...